docker/init.sls

```diff
@@ -66,5 +66,17 @@ docker-service:
     - enable: True
     {% if pkg and "process_signature" in pkg %}
     - sig: {{ pkg.process_signature }}
-    {% endif -%}
+    {% endif %}
 
+python-pip:
```

Comment:
This is almost surely going to cause a id conflict. Please use a more unique id and the name option.